ui-components/src/navigation/ProgressNav.scss

```diff
@@ -58,45 +74,79 @@
 
   &:last-of-type {
     &:after {
-      @apply left-auto;
+      left: auto;
       right: 50%;
     }
   }
 
+  a {
+    padding-top: var(--dot-label-padding-top);
+    padding-left: var(--dot-label-padding-left);
+    color: var(--completed-step-font-color);
+    display: block;
+    position: relative;
+    z-index: 3;
+  }
+
   &.is-active {
     &:before {
-      @apply h-3;
-      @apply w-3;
-      @apply bg-primary;
-      @apply top-0;
+      background-color: var(--active-step-color);
     }
 
     a {
-      @apply text-black;
-      @apply font-bold;
+      color: var(--active-step-font-color);
+      font-weight: var(--dot-active-font-weight);
     }
   }
 
   &.is-disabled {
     &:before {
-      @apply h-3;
-      @apply w-3;
-      @apply bg-gray-450;
-      @apply top-0;
+      background-color: var(--future-step-color);
     }
 
     a {
-      @apply pointer-events-none;
-      @apply cursor-default;
+      color: var(--future-step-font-color);
+      pointer-events: none;
+      cursor: default;
     }
   }
+}
+
+.progress-nav__bar {
+  border-spacing: var(--bar-spacing);
+  table-layout: fixed;
+}
 
+.progress-nav__bar-item {
+  // drop the labels on mobile view
+  font-size: 0;
+  padding-top: var(--bar-label-padding-top);
+  padding-left: var(--bar-label-padding-left);
+  position: relative;
+  display: table-cell;
+  white-space: nowrap;
+  float: none;
+  max-width: 20%;
```

Comment:
What is this max-width used for? If I remove it I don't notice any difference

Reply:
I am not sure and also didn't notice any differences through testing. Looks like this was added in the original version of the component almost 2 years ago 😨 . I removed it but do you happen to remember the intention behind this @slowbot?